docs/core/tutorials/nft-tutorials/whitelists.md

```diff
@@ -0,0 +1,308 @@
+---
+title: Merkle Tree Whitelist
+displayed_sidebar: coreSidebar
+---
+
+# Merkle Tree NFT Whitelist on CoreSpace using Hardhat
+
+### Requirements
```

Comment:
The requirements seems strange here. These requirements can be part of env setup